### DISCUSSION TITLE
This is so terribly wrong!

### DISCUSSION THREAD

**Discussion by k16e** [tailwindlabs/tailwindcss#4377] [General]
Hi, look at the image...how, you can't place `padding: 0 !important` on the `button` element! Like, I'm literally tearing my hair out trying to get my button padded... Please fix this out ASAP. Thanks
![image](https://user-images.githubusercontent.com/52034326/118518268-dc29b980-b72f-11eb-9a86-de5a698eb611.png)

**Comment by k16e** [comment]
I noticed this because I wasn't purging correctly. But I shudder to think the reset styles went down that level...or what am I ignorant of?

**Comment by dance2die** [comment]
Tailwind CSS's preflight reset doesn't add any `!important`: https://github.com/tailwindlabs/tailwindcss/blob/master/src/plugins/css/preflight.css

Do you know where `input-css-AojJe8` line185 is coming from? An external 3rd party UI component?

**Reply by k16e** [reply]
I see. Made me quite furious. I don't see anywhere else it might be coming from. As soon as I enable purge

```
purge: [
    './dist/**/*.html',
    './**/*.{js,jsx,ts,tsx,vue}'
]
```
it comes up! https://gitlab.com/k16e/e-adr-v1
